Fails to compile

Hi, I'm getting the following errors 

` Compiling ego v1.1.2
error[E0599]: no variant or associated item named `DisableVersion` found for enum `AppSettings` in the current scope
  --> /home/fabio/.cargo/registry/src/github.com-1ecc6299db9ec823/ego-1.1.2/src/cli.rs:23:31
   |
23 |         .setting(AppSettings::DisableVersion)
   |                               ^^^^^^^^^^^^^^
   |                               |
   |                               variant or associated item not found in `AppSettings`
   |                               help: there is a variant with a similar name: `DisableVersionFlag`

error[E0599]: no variant or associated item named `ColoredHelp` found for enum `AppSettings` in the current scope
  --> /home/fabio/.cargo/registry/src/github.com-1ecc6299db9ec823/ego-1.1.2/src/cli.rs:24:31
   |
24 |         .setting(AppSettings::ColoredHelp)
   |                               ^^^^^^^^^^^ variant or associated item not found in `AppSettings`

For more information about this error, try `rustc --explain E0599`.
error: failed to compile `ego v1.1.2`, intermediate artifacts can be found at `/tmp/cargo-installYiEjMx`

Caused by:
  could not compile `ego` due to 2 previous errors
`
I'm using rust 1.55.0 on gentoo since the stable version was giving errors during clap compile.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,7 +18,7 @@ categories = ["command-line-utilities"]
 users = "0.11.0"
 simple-error = "0.2.3"
 posix-acl = "1.0.0"
-clap = "3.0.0-beta.2"
+clap = "=3.0.0-beta.2"
 log = { version = "0.4.14", features = ["std"] }
 ansi_term = "0.12.1"
 shell-words = "1.0.0"
@@ -28,4 +28,4 @@ default = []
 update-snapshots = []
 
 [dev-dependencies]
-clap_generate = "3.0.0-beta.2"
+clap_generate = "=3.0.0-beta.2"

